apps/api-extractor/src/generators/ApiModelGenerator.ts

```diff
@@ -765,6 +770,7 @@ export class ApiModelGenerator {
       const releaseTag: ReleaseTag = apiItemMetadata.effectiveReleaseTag;
       const isOptional: boolean =
         (astDeclaration.astSymbol.followedSymbol.flags & ts.SymbolFlags.Optional) !== 0;
+      const isReadonly: boolean = this._determineReadonly(astDeclaration);
```

Comment:
Same as above, I don't believe method signatures can be readonly. https://www.typescriptlang.org/play?#code/MYGwhgzhAECiAeYC2AHEBTaBvAUNf0ATumACYD2AdiAJ7QBm55AFAJQBc0AbuQJanYAvjmE5elAC7pC9MMEwJkadACZseAsTJVaDJm049+AbhFA